test/onnx/onnx_test.cpp

```diff
@@ -699,8 +699,8 @@ TEST_CASE(add_scalar_test)
 {
     migraphx::program p;
     auto l0 = p.add_parameter("0", migraphx::shape{migraphx::shape::float_type, {2, 3, 4, 5}});
-    auto l1 =
-        p.add_literal(migraphx::literal{migraphx::shape{migraphx::shape::float_type, {1}}, {1}});
+    auto l1 = p.add_literal(
+        migraphx::literal{migraphx::shape{migraphx::shape::float_type, {1}, {0}}, {1}});
```

Comment:
The previous implementation for parsing different data types isn't being tested either according to codecov. I think this is something we need to take care of, but it can probably be addressed in a different PR.